tasks/install.yml

```diff
@@ -40,10 +40,17 @@
   tags:
     - sdkman_privilege
 
-# Alpine Linux requires GLIBC
 - block:
+    # Alpine Linux requires GLIBC
     - name: Install GLIBC for Alpine Linux
       include_tasks: install_glibc_alpine.yml
+
+    - name: Install update-alternatives dependencies for Alpine Linux
```

Comment:
I wonder if this shouldn't just be part of the rest of system package installation. At least I can't think of a reason why it shouldn't just be grouped with the rest of them. Thoughts?

Reply:
My rationale is that this is an opt-in feature not directly tied to the core SDKMAN functionality -if the user doesn't need this, the additional package doesn't need to be installed and the rest of the role works as expected. If they're using Alpine especially, keeping the image size small by avoiding unnecessary dependencies might be a concern.

But I think the effect may ultimately be negligible, its not a big package and it'll certainly keep things simpler to include it in the system packages. Let me know what you think! I'm perfectly fine moving it to system packages.

Reply:
Yeah... let's keep it simple and move it to the `system_packages`. If folks start complaining about a bloated Alpine image, we can reconsider.